apps/nova/src/components/dashboard/ChatSummaryWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import { Route } from "@/types/routes";
+import { useFindUserRooms } from "@litespace/headless/chat";
+import { useUser } from "@litespace/headless/context/user";
+import { ChatSummary, type ChatSummaryProps } from "@litespace/luna/Chat";
+import { orUndefined } from "@litespace/sol/utils";
+import { IRoom } from "@litespace/types";
+import dayjs from "dayjs";
+import { useMemo } from "react";
+
+function organizeRooms(
+  list: IRoom.FindUserRoomsApiRecord[] | null
+): ChatSummaryProps["rooms"] {
+  return (
+    list?.map((room) => ({
+      id: room.roomId,
+      url: Route.Chat.concat("?room=", room.roomId.toString()),
+      name: orUndefined(room.otherMember?.name),
+      image: orUndefined(room.otherMember?.image),
+      message: room.latestMessage?.text || "TODO",
+      sentAt: room.latestMessage?.updatedAt || dayjs().toString(), // TODO
+      read: room.unreadMessagesCount === 0,
+    })) || []
+  );
+}
+
+export const ChatSummaryWrapper = () => {
```

Comment:
I am not a fan of the word `...Wrapper`. I believe we can do better. We can just name it `ChatSummary`. For the original `ChatSummary` we can just call it the `Summary`. 

```tsx
import {ChatSummary as Summary} from '@litespace/luna/Chat'
export const ChatSummary = () => {
    return <Sumary />
}
```

This also applies to the file names and other components located in `components/dashboard/*`.

Reply:
Nice solution 🚀